File: C#_Projects/Okul_Bolumleri/Adalet.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Okul_Bolumleri
{
    public class Adalet : Anasayfa
    {
        double bolumUcreti = 20000;
        int bolumKontenjani = 71;

        public override void bolumOzellikleriniGetir()
        {
            Console.WriteLine($"Bu bölümün ücreti = {bolumUcreti} TL, maksimum öğrenci kontenjanı ise {bolumKontenjani} kişidir.");
        }

        public override void bursluFiyatiHesapla()
        {
            bursluFiyat = bolumUcreti - (bolumUcreti * (bursOrani / 100));

            if (bursOrani > 0 && bursOrani <= 100)
            {
                Console.WriteLine($"Bu bölümün % { bursOrani} burslu hali = { bursluFiyat} TL'dir.");
            }
            else
            {
                Console.WriteLine("Burs oranı limiti aşılıyor.");
            }
        }

    }
}

File: Javascript_Projects/Universite_Bolumleri/adalet.js
import { anasayfa } from "./anasayfa.js";


export class adalet extends anasayfa {
    constructor(bursOrani) {
        super(bursOrani);
    }
    bolumUcreti = 20000;
    bolumKontenjani = 71;


    bolum_ozelliklerini_getir() {
        console.log(`Adalet bölümünün üniversite ücreti ${this.bolumUcreti} TL, maksimum öğrenci kontenjanı ise ${this.bolumKontenjani} kişidir`);

    }
    burslu_fiyati_hesapla() {
        this.bursluFiyat = this.bolumUcreti - (this.bolumUcreti * (this.bursOrani / 100));


        if (this.bursOrani > 0 && this.bursOrani <= 100) {
            console.log(`Bu bölümün burs orani % ${this.bursOrani}, burslu fiyatı ise ${this.bursluFiyat} TL'dir`);
        }
        else {
            console.log("Burs orani limiti aşılıyor");
        }
    }


}